pkg/controller/catalogsourceconfig/catalogsourceconfig_controller.go

```diff
@@ -100,6 +103,7 @@ func (r *ReconcileCatalogSourceConfig) Reconcile(request reconcile.Request) (rec
 	instance.EnsureGVK()
 
 	err = r.CatalogSourceConfigHandler.Handle(context.TODO(), instance)
+	status.SendSyncMessage(err)
```

Comment:
How are you differentiating between a sync (event was received) and a successful sync (event was handled without errors) in the case where `err` is `nil`? This seems like you are counting a "successful sync" also as a sync.

Reply:
The code treats both a sync event and a successful sync in the same manner. If you think we should treat them as separate events could you explain what you think that should look like?